app/src/main/java/br/org/cesar/discordtime/stickysessions/ui/session/SessionActivity.java

```diff
@@ -110,6 +110,13 @@ private void configureSession() {
         }
     }
 
+    @Override
+    protected void onNewIntent(Intent intent) {
+        super.onNewIntent(intent);
+        setIntent(intent);
+        configureSession();
```

Comment:
Please clear notes adapter before configuring next session, since an error could cause the previous list to be shown.